java/pom.xml

```diff
@@ -155,8 +168,12 @@
                 </executions>
                 <configuration>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
-                    <forkMode>once</forkMode>
+
+                    <!-- It's important to run tests serially because we are doing timing tests in some of the tests. -->
```

Comment:
**This is important!**  *The tests can fail when not run serially.*  This may indicate a thread-safety issue in the Java *OR* C code.  This might be worth investigating.